databroker/broker.py

```diff
@@ -575,6 +545,68 @@ def get_events(self, header, pvs):
                 yield Document('Event', doc)
 
 
+class DangerBroker(Broker):
```

Comment:
See the warning in the docstring to the method `move_files_for_run`, below. But, @tacaswell, maybe your snarky name should come with a one-sentence explanation in the top-level docstring.